crates/astria-sequencer/src/accounts/state_ext.rs

```diff
@@ -49,12 +48,11 @@ impl<'a> std::fmt::Display for StorageKey<'a> {
         Ok(())
     }
 }
-fn balance_storage_key(address: Address, asset: asset::Id) -> String {
-    format!(
-        "{}/balance/{}",
-        StorageKey(&address),
-        asset.encode_hex::<String>()
-    )
+fn balance_storage_key<TAsset: Into<asset::IbcPrefixed>>(
+    address: Address,
+    asset: TAsset,
+) -> String {
+    format!("{}/balance/{}", StorageKey(&address), asset.into(),)
```

Comment:
i would prefer the storage keys continue to use the 32 byte asset ID - adding the `ibc/` prefix in front of every asset ID seems unnecessary and just adds extra data to storage, i don't really see a benefit to adding the prefix and it introduces ibc-specific naming conventions into the internal db which seems weird

Reply:
I have addressed this (and also all other instances where assets are used) in e968da4